src/EditorFeatures/CSharpTest/Completion/CompletionProviders/AbstractCSharpCompletionProviderTests.cs

```diff
@@ -31,32 +31,32 @@ protected override TestWorkspace CreateWorkspace(string fileContents)
             string code, int position,
             string expectedItemOrNull, string expectedDescriptionOrNull,
             SourceCodeKind sourceCodeKind, bool usePreviousCharAsTrigger, bool checkForAbsence,
-            int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix)
+            int? glyph, int? matchPriority, bool? hasSuggestionItem, string displayTextSuffix, string inlineDescription = null)
```

Comment:
with this many input, probably it is time to add TestOption object or something :)

Reply:
Yeah. 
FYI @dpoeschl , we probabaly should do this if both PR got approved.

Reply:
Yep, I'm about to make it worse too. :)

Reply:
it would probably be a good idea to have this.  But i would recommend doing it in another PR first to keep this PR's change small.  up to you if you would want to front or rear load that work.